src/metabase/driver/sql_jdbc.clj

```diff
@@ -171,6 +171,16 @@
                                :dialect (sql.qp/quote-style driver)))]
     (qp.writeback/execute-write-sql! db-id sql)))
 
+(defmethod driver/add-columns! :sql-jdbc
```

Comment:
Why is this needed now?

Reply:
It looks like this got copied from above, probably by git while merging. I've removed it 2fe2875b48c0a197c30eb275f9b9e9533e60d431